lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -1240,6 +1283,8 @@ def test_basic(self, mock_emit):
             "endorsed_comment_list_url": None,
             "non_endorsed_comment_list_url": None,
             "editable_fields": ["abuse_flagged", "following", "raw_body", "title", "topic_id", "type", "voted"],
+            'read': False,
+            'has_endorsed': False
```

Comment:
Endorsement is used for both question and discussion threads, and the term "answered" is only appropriate for the former. For a bit more context, the notion of an endorsed response actually originated before my time (and, therefore, before the question/discussion separation). We just treat endorsed responses in question threads specially.